patches/minecraft/net/minecraft/block/BlockChorusFlower.java.patch

```diff
@@ -0,0 +1,204 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockChorusFlower.java
++++ ../src-work/minecraft/net/minecraft/block/BlockChorusFlower.java
+@@ -17,6 +17,9 @@
+ import net.minecraft.util.EnumFacing;
+ import net.minecraft.util.math.BlockPos;
+ import net.minecraft.world.World;
++import net.minecraftforge.common.MinecraftForge;
++import net.minecraftforge.event.world.BlockEvent;
++import net.minecraftforge.fml.common.eventhandler.Event;
```

Comment:
From [CONTRIBUTING.md](https://github.com/MinecraftForge/MinecraftForge/blob/1.10.x/CONTRIBUTING.md) (which appears at the top when making an issue/PR), there is a link leading [here](https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge#conventions-for-coding-patches-for-a-minecraft-class-javapatch) which mentions said style guideline.